crates/sui-indexer/src/models/transactions.rs

```diff
@@ -1,19 +1,20 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::errors::IndexerError;
-use crate::schema::transactions;
-use crate::schema::transactions::transaction_digest;
-use crate::types::SuiTransactionFullResponse;
-use crate::PgPoolConnection;
-
 use diesel::prelude::*;
 use diesel::result::Error;
+
 use sui_json_rpc_types::{
     OwnedObjectRef, SuiObjectRef, SuiTransaction, SuiTransactionDataAPI, SuiTransactionEffects,
     SuiTransactionEffectsAPI,
 };
 
+use crate::errors::IndexerError;
+use crate::schema::transactions;
+use crate::schema::transactions::transaction_digest;
+use crate::types::SuiTransactionFullResponse;
+use crate::PgPoolConnection;
```

Comment:
question: is this just the result of some formatter? Looks like L4-9 were just moved to L12-16

Reply:
just re-org the `use`, the idiomatic order is std -> external -> crate, I put sui / Mysten ones between external and crate.